qldbdriver/result_test.go

```diff
@@ -106,15 +106,15 @@ func TestResult(t *testing.T) {
 
 				// Default page
 				assert.True(t, result.Next(&transactionExecutor{nil, nil}))
-				assert.Equal(t, new(int64), result.statementStatistics.timingInformation.GetProcessingTimeMilliseconds())
-				assert.Equal(t, new(int64), result.statementStatistics.ioUsage.GetReadIOs())
-				assert.Equal(t, new(int64), result.statementStatistics.ioUsage.getWriteIOs())
+				assert.Equal(t, int64(0), *result.metrics.timingInformation.GetProcessingTimeMilliseconds())
```

Comment:
Why can't this just be `0`?

Reply:
Because `0` by itself is of type `int` and GetProcessingTimeMilliseconds returns value of type `int64`.

The assertion fails and returns:
```
Expected :int(0)
Actual   :int64(0)
```

Reply:
Ok cool